proto/teleport/lib/teleterm/v1/cluster.proto

```diff
@@ -58,6 +58,16 @@ message Cluster {
   // ProxyVersion is the cluster proxy's service version.
   // Only present when detailed information is queried from the proxy server.
   string proxy_version = 10;
+  // show_resources tells if the cluster can show requestable resources on the resources page.
+  // Controlled by the cluster config.
+  ShowResources show_resources = 11;
+}
+
+// ShowResources tells if the cluster can show requestable resources on the resources page.
+enum ShowResources {
+  SHOW_RESOURCES_UNSPECIFIED = 0;
+  SHOW_RESOURCES_ACCESSIBLE_ONLY = 1;
+  SHOW_RESOURCES_REQUESTABLE = 2;
```

Comment:
Nit, but I'd make requestable 1 and accessible only 2, only because requestable is generally used over accessible only.

I wanted to check if it follows how it's defined in the api protos, but `api/proto/teleport/legacy/types/types.proto` defines it as as string anyway.